ajax/libs/lethargy/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "lethargy",
+  "filename": "lethargy.min.js",
+  "version": "1.0.4",
+  "description": "Distinguish between scroll events initiated by the user, and those by inertial scrolling",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/d4nyll/lethargy.git"
+  },
+  "keywords": [
+    "scroll",
+    "wheel",
+    "mouse",
+    "mousewheel",
+    "scrolljacking",
+    "inertial",
+    "trackpad",
+    "magic",
+    "mouse"
+  ],
+  "author": "Daniel Li <dan@danyll.com>",
+  "license": "MIT",
+  "npmName": "lethargy",
+  "npmFileMap": [
+    {
+      "basePath": "",
+      "files": [
+        "lethargy*.+(js|css|map)"
```

Comment:
Because you don't know if there will be `.css` file in the future. `lethargy*` will get something we don't need.